client/payment-details/dispute-details/dispute-resolution-footer.tsx

```diff
@@ -0,0 +1,285 @@
+					<Button
+						variant="secondary"
+						onClick={ () => {
+							wcpayTracks.recordEvent(
+								wcpayTracks.events
+									.PAYMENT_DETAILS_VIEW_DISPUTE_EVIDENCE_BUTTON_CLICK,
+								{
+									dispute_status: dispute.status,
+								}
+							);
+							const challengeUrl = getAdminUrl( {
+								page: 'wc-admin',
+								path: '/payments/disputes/challenge',
+								id: dispute?.id,
+							} );
+							getHistory().push( challengeUrl );
+						} }
+					>
+						{ __(
+							'View submitted evidence',
+							'woocommerce-payments'
+						) }
+					</Button>
+				</FlexItem>
+			</Flex>
+		</CardFooter>
+	);
+};
+
+const DisputeWonFooter: React.FC< {
+	dispute: Dispute;
+} > = ( { dispute } ) => {
+	const closedDateFormatted = dispute?.metadata.__dispute_closed_at
+		? dateI18n(
+				'M j, Y',
+				moment
+					.unix(
+						parseInt( dispute.metadata.__dispute_closed_at, 10 )
+					)
+					.toISOString()
+		  )
+		: '-';
+
+	return (
+		<CardFooter className="transaction-details-dispute-footer">
+			<Flex justify="space-between">
+				<FlexItem>
+					{ createInterpolateElement(
+						sprintf(
+							/* Translators: %s - formatted date, <a> - link to documentation page */
+							__(
+								'Good news! You won this dispute on %s. The disputed amount and the dispute fee have been credited back to your account. <a>Learn more about preventing disputes</a>.',
+								'woocommerce-payments'
+							),
+							closedDateFormatted
+						),
+						{
+							a: (
+								// eslint-disable-next-line jsx-a11y/anchor-has-content -- Link content is provided by createInterpolateElement
+								<a
+									target="_blank"
+									rel="noopener noreferrer"
+									href="https://woocommerce.com/document/woopayments/fraud-and-disputes/"
+								/>
+							),
+						}
+					) }
+				</FlexItem>
+				<FlexItem className="transaction-details-dispute-footer__actions">
+					<Button
+						variant="secondary"
+						onClick={ () => {
+							wcpayTracks.recordEvent(
+								wcpayTracks.events
+									.PAYMENT_DETAILS_VIEW_DISPUTE_EVIDENCE_BUTTON_CLICK,
+								{
+									dispute_status: dispute.status,
+								}
+							);
+							const challengeUrl = getAdminUrl( {
+								page: 'wc-admin',
+								path: '/payments/disputes/challenge',
+								id: dispute?.id,
+							} );
+							getHistory().push( challengeUrl );
+						} }
+					>
+						{ __( 'View dispute details', 'woocommerce-payments' ) }
+					</Button>
+				</FlexItem>
+			</Flex>
+		</CardFooter>
+	);
+};
+
+const DisputeLostFooter: React.FC< {
+	dispute: Dispute;
+} > = ( { dispute } ) => {
+	const isSubmitted = !! dispute?.metadata.__evidence_submitted_at;
+	const isAccepted = dispute?.metadata.__closed_by_merchant === '1';
+	const disputeFee = dispute && getDisputeFee( dispute );
+	const disputeFeeFormatted = disputeFee
+		? formatExplicitCurrency( disputeFee.fee, disputeFee.currency )
+		: '-';
```

Comment:
Would a `getFormattedDisputeFee()` util function be useful?

Reply:
Great idea, I've added this in 582655a